src/script_opt/ZAM/ZInst.cc

```diff
@@ -26,7 +26,7 @@ void ZInst::Dump(FILE* f, const string& prefix, const string& id1, const string&
                  const string& id4) const {
     fprintf(f, "%s ", ZOP_name(op));
     // fprintf(f, "(%s) ", op_type_name(op_type));
-    if ( t && 0 )
+    if ( t && false )
```

Comment:
:man_shrugging: - this seems like it should be some environment variable or setting...

Reply:
Fine to remove